system/Database/SQLSRV/PreparedQuery.php

```diff
@@ -92,7 +83,7 @@ public function _execute(array $data): bool
      */
```

Comment:
```php
    /**
     * Returns the result object for the prepared query.
     *
     * @return mixed
     */
    public function _getResult()
    {
        return $this->statement;
    }
```

Should return a statement resource or false on failure if $this->statement is the result of sqlsrv_prepare()